api/v1/db/oracledb/games-dao.js

```diff
@@ -112,4 +112,28 @@ const postGame = async (body) => {
   }
 };
 
-module.exports = { getGames, getGameById, postGame };
+/**
+ *
+ * @param {id} developerId id of developer record to check existance
+ * @returns true if developer record is found and false if no records are found
+ */
+const isValidDeveloper = async (developerId) => {
+  const sqlParams = { id: developerId };
+  const sqlQuery = `
+    SELECT ID AS "id"
```

Comment:
We usually use `COUNT` in SQL to make sure the records with certain criteria exist.